hoot-core-test/src/test/cpp/hoot/core/ops/CalculateStatsOpTest.cpp

```diff
@@ -111,10 +106,9 @@ class CalculateStatsOpTest : public CppUnit::TestFixture
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Conflatable Features"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Untagged Feature Count"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Unconflatable Features"));
-    CPPUNIT_ASSERT_EQUAL(12.0, calcStatsOp->getSingleStat("Total Features Processed By Conflation"));
     CPPUNIT_ASSERT_EQUAL(12.0, calcStatsOp->getSingleStat("Total Conflated Features"));
     CPPUNIT_ASSERT_EQUAL(19.0, calcStatsOp->getSingleStat("Total Unmatched Features"));
-    CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Features Marked for Review"));
+    CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Features Involved in a Review"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Number of Reviews to be Made"));
```

Comment:
The UI code uses these titles to correctly parse a subset of the stats data for display.  I don't think "Marked for Review" is such a bad term.  To me it clearly signifies number of features that could not be determined to be a match or a miss and must be reviewed.  Unless you feel strongly I'd rather not make a change that has to propagate to the UI code.